services/horizon/internal/test/integration/integration.go

```diff
@@ -602,11 +638,32 @@ func (i *Test) CreateSignedTransaction(
 	return tx, nil
 }
 
+func (i *Test) CloseCoreLedger() error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	return i.cclient.ManualClose(ctx)
```

Comment:
Logging can be helpful with debugging.
```suggestion
	i.t.Log("Closing ledger manually...")
	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
	defer cancel()
	return i.cclient.ManualClose(ctx)
```

Reply:
Note taken, thanks.